### PR TITLE
Korean translation polishment

### PR DESCRIPTION
**Pull request by jerrybeomsoo**
- Polished existing Korean translation which could be misleading
- Translated several strings that were not previously in Korean

### CODE REVIEW

File: public/src/js/strings.js [hunk 1]
```diff
@@ -156,14 +156,14 @@ var translations = {
 		en: "Extreme",
 		cn: "魔王",
 		tw: "魔王",
-		ko: "귀신"
+		ko: "오니"
```
Comment (LoveEevee) [inline public/src/js/strings.js:159]: The Extreme difficulty in the korean releases of taiko on switch and ps4 is called 귀신, only official names are allowed so please change this back

File: public/src/js/strings.js [hunk 2]
```diff
@@ -1088,23 +1101,23 @@ var translations = {
...
+			ko: "제작자:"
 		}
 	},
 	withLyrics: {
 		ja: "歌詞あり",
 		en: "With lyrics",
 		cn: "带歌词",
 		tw: "帶歌詞",
-		ko: "가사가있는"
+		ko: "가사 포함됨"
```
Comment (LoveEevee) [inline public/src/js/strings.js:1112]: Google tells me 포함됨 is a typo and should be 포함된 instead

**Review by LoveEevee — CHANGES_REQUESTED** [review]

File: public/src/js/strings.js (hunk 1 above)
Comment (jerrybeomsoo) [reply public/src/js/strings.js]: Reverted in commit.

File: public/src/js/strings.js (hunk 2 above)
Comment (jerrybeomsoo) [reply public/src/js/strings.js:1112]: It is actually not a typo nor gramatically incorrect, and "포함됨" is more natural.

**Review by LoveEevee — APPROVED** [review]

**Comment by bui** [comment]
Thanks!

**Pull request merged**